app/src/components/Router.js

```diff
@@ -0,0 +1,22 @@
+import { BrowserRouter,Routes,Route } from "react-router-dom";
+import Navbar from "./Navbar";
+import App from "../App"
+import About from "../pages/About"
+import FAQ from "../pages/FAQ";
+
+export default function Router ()
+{
+    return(
+    <BrowserRouter>
+    <Routes>
```

Comment:
The layout of the main page has all the content in one page. This type of routing might be best suited when we add the admin to the project.

Reply:
You mean to use anchors as in the previous version?

Reply:
Yes anchor tags since all the content is in one page